src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    Truck truck = new Truck();
+    Bulldozer bulldozer = new Bulldozer();
+    Excavator excavator = new Excavator();
+    Machine[] machines = new Machine[]{truck, bulldozer, excavator};
 
+    public void values() {
+        for (Machine values : machines){
+            values.doWork();
+            values.stopWork();
+        }
+    }
```

Comment:
The `values` method is defined but not called anywhere in the `MainApp` class. Consider adding a `main` method to call `values` and execute the operations on the machines.